source/dapp.rst

```diff
@@ -3,10 +3,26 @@
 Credits Dapp
 ============
 
-Credits Dapp is a packaged Python module code that provides a
+Credits Dapp is a distributed blockchain-based application developed using
+the Credits Core framework. Dapp is a unit of business logic, and it's
+supposed to be designed with specific real life use case in mind.
+
+Credits Dapp is a software module designed to run as part of the Credits
```

Comment:
This is where python module and more generic "module" as a distinct piece of software and functionality collide. Yes, we pick and choose from modules, but that forms the Dapp, a finished set of functionality. It has to be named something. If not "module" then offer a synonym.